webapp/src/test/java/org/apache/falcon/resource/ExtensionManagerIT.java

```diff
@@ -100,8 +100,6 @@ public void testTrustedExtensionJob() throws Exception {
         // validate job list results
         ExtensionJobList jobs = context.getExtensionJobs("hdfs-mirroring", null, null, null, null, null);
         Assert.assertEquals(jobs.numJobs, 2);
-        Assert.assertEquals(jobs.job.get(0).jobName, JOB_NAME_1);
```

Comment:
Can we unable this test case and see if test cases are running correctly.Right now the test cases have been disabled.

@Test (enabled = false)